app/cdap/components/PipelineDiff/DiffInfo/DiffInfoTable.tsx

```diff
@@ -23,56 +23,164 @@ import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import TableCell from '@material-ui/core/TableCell';
 import TableContainer from '@material-ui/core/TableContainer';
-import { DiffIndicator, IPipelineStage } from '../types';
+import { DeepPartial, DiffIndicator, IPipelineStage } from '../types';
 import { ADDED_REGEX, DELETED_REGEX, I18N_PREFIX } from '../constants';
 
-const DiffInfoTableContainer = styled(TableContainer)`
-  flex-grow: 1;
-  max-width: 100%;
-  width: 100%;
-`;
-
-function getEntryData(diffIndicator: DiffIndicator, propertyName: string, propertyValue) {
-  if (diffIndicator === DiffIndicator.DELETED || diffIndicator === DiffIndicator.ADDED) {
-    return { name: propertyName, value: propertyValue as string };
+function _flattenObject<T>(obj: T, name: string = '') {
```

Comment:
if this is only used for naming can we rename it getName or something like that?